src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -91,3 +91,44 @@ function *(A::DerivativeOperator{T,N},M::AbstractArray{T}) where {T<:Real,N}
     LinearAlgebra.mul!(x_temp, A, M)
     return x_temp
 end
+
+function LinearAlgebra.ldiv!(M_temp::AbstractArray{T,MT}, A::DerivativeOperator{T,N}, M::AbstractArray{T,MT}) where {T<:Real, N, MT}
```

Comment:
Could I suggest a change throughout this function? It will improve memory performance if you concretize the operators to SparseArrays, as memory can be quite a major limitation on your domain size. 

If A and B have N^2 elements each, Kron(A,B) will have N^4. But since A and B are in this case sparse Multidiagonal,  if you use this kind of concretization you can reduce the memory consumption of kron(A,B) to O(N^2) since they have O(N) elements in sparse form.

Reply:
Yes, that is what I will go towards.

Reply:
I think as a first pass to get tests working it's fine to have it concrete, but I agree that in any case this will need to be sparse.